cuda_core/cuda/core/experimental/_launch_config.pxd

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+cdef bint _inited
+cdef bint _use_ex
+
+cdef void _lazy_init() except *
+
+cdef class LaunchConfig:
+    """Customizable launch options."""
+    cdef public tuple grid
+    cdef public tuple cluster
+    cdef public tuple block
```

Comment:
IIUC, these are all 3-tuples of ints.  Those are pretty expensive, particularly if used from other C or Cython code.  Could we make these `int[3]` instead?  (And of course the updates to `convert_to_3_tuple`...)